### PR TITLE
added required methods and some changes for python 3 compatability

### PR DESCRIPTION
**Pull request by 13bscsaamjad**
- `class_prop` are not working well with sardana3; changed to `ctrl_properties` which is better for having different properties for different equipment.
- added required `PreStartAll` and `StartOne` methods for sardana 3.
- remove CT suffix from methods which does not seems to work with sardana 3.

### CODE REVIEW

File: sardana_adlink/ctrl/AdlinkAICoTiCtrl.py [hunk 1]
```diff
@@ -232,15 +232,21 @@ def LoadOne(self, axis, value, repetitions, latency):
         self.AIDevice["NumOfTriggers"] = self._repetitions
         self.AIDevice['ChannelSamplesPerTrigger'] = chn_samp_per_trigger
 
+    def PreStartAll(self):
```
Comment (reszelaz) [inline sardana_adlink/ctrl/AdlinkAICoTiCtrl.py]: I think this is not mandatory to re-implement it.

File: sardana_adlink/ctrl/AdlinkAICoTiCtrl.py [hunk 2]
```diff
@@ -232,15 +232,21 @@ def LoadOne(self, axis, value, repetitions, latency):
...
+        pass
+
     def PreStartOne(self, axis, value=None):
         if axis != 1:
             self._master_channel = axis
         return True
+    
+    def StartOne(self, axis, value=None):
```
Comment (reszelaz) [inline sardana_adlink/ctrl/AdlinkAICoTiCtrl.py]: Sorry this is bug in Sardana that I introduced. It whould not be mandatory to implement it - see https://github.com/sardana-org/sardana/pull/1403.

**Review by reszelaz — CHANGES_REQUESTED** [review]
Many thanks Abdullah for the changes! Thanks to this PR I discovered that I wrongly made the `StartOne()` mandatory. Could you check my comments and make the necessary changes?

**Comment by 13bscsaamjad** [comment]
Thanks for the review. I have now removed `StartOne` and `PreStartAll`.

**Review by rhomspuron — CHANGES_REQUESTED** [review]
Could you remove this change? We will do it on the integration.

**Comment by 13bscsaamjad** [comment]
@rhomspuron sorry, which one?

**Review by 13bscsaamjad — COMMENTED** [review]
reviewed

**Comment by rhomspuron** [comment]
Hi @13bscsaamjad, 

Sorry I did not select the correct file. Could you revert the bump version commit? We use the bumpversion: https://pypi.org/project/bumpversion/, to change the version. It changes two files at the same time: setup.py and .bumpversion.cfg, and creates a tag for the version. We recommend to not change by hand the version, we will do it  after to integrate the PR.

Thanks,
Roberto

**Pull request merged**